content/maxwell3_fdem/emi_tutorial/induced_currents_body.rst

```diff
@@ -0,0 +1,99 @@
+.. _induced_currents_body:
+
+Induced currents in the body
+============================
+
+.. topic:: Purpose
+  XXX
+
+Consider a simple equivalent circuit as shown in :numref:`Concepts_3loops_only`.
+
+.. figure:: ./images/Concepts_3loops_only.png
+   :align: center
+   :scale: 50%
+   :name: Concepts_3loops_only
+
+   Conceptual diagram for 3-loops system.
+
+
+A time-varying current is put through the Tx (loop 1), which then generates a time varying magnetic flux (primary magnetic flux), which we denote :math:`\Phi^{p}`. If coupled, this will induce currents in the body (Loop2) according to :ref:`Lenz's Law<lenz_law>`. We can examine the nature of the induced current by treating the loop as a Resistor-
```

Comment:
"If coupled, this will induce..." In the introduction, one of the goals is to discuss coupling but until you get to that point, maybe remove the mention to it here? My first thought was, ok, what is coupling and are things coupled in the diagram?
Could rewrite as "This can generate a time-varying magnetic flux..." or hyperlink coupling to the appropriate page?